app/services/ncr/work_order_value_normalizer.rb

```diff
@@ -41,6 +42,12 @@ def normalize_soc_code
       end
     end
 
+    def normalize_approving_official_email
+      if !work_order.approving_official_email.present? && work_order.approvers.any?
+        work_order.approving_official_email = work_order.approvers.first.try(:email_address)
```

Comment:
It appears to be necessary in order for `def approver_changed?` to be effective, which is called from the controller to signal that some model attributes have changed. It seems to essentially be a shorthand for indicating that some association has been correctly configured.

If you feel that merits a tech debt card, feel free to open one.